placeholder-projects/buildSrc/src/main/kotlin/Build.kt

```diff
@@ -90,32 +102,35 @@ object Build {
       .replace(REACT_REGEX, "\n")
       .replace(DEPENDENCY_REGEX, "\n")
 
-  private fun noArgsFunctionCallRegex(fName: String) = Regex("$fName[ \\t]*[(][ \\t]*[)]")
+  private fun noArgsFunctionCallRegex(fName: String) = Regex("$fName${WS}[(]${WS}[)]")
 
   private fun getInputFile() = getEnv(envName = "INPUT_FILE")?.let(::File)
     ?.takeIf { !it.isDirectory && it.exists() }?.absolutePath
 
-  fun Project.isRelevantProject(): Boolean {
-    val inputScript = getInputFile()?.let(::File)?.readText() ?: return false
-
+  private fun Project.isRelevantProject(inputScript: String): Boolean {
     val projectName = determinePlaceholderProject(inputScript)
     return projectName == name
   }
 
-  private fun Project.manageInputFile(sourceSet: String, buildStatus: BuildStatus) {
-    val inputScript = getInputFile()?.let(::File)?.readText() ?: return
-    buildStatus.dependencies += extractAllDependencies(inputScript)
+  private fun Project.manageInputFile(inputScript: String, sourceSet: String) {
     val placeholderFile = layout.projectDirectory.file("src/$sourceSet/kotlin/Script.kt").asFile
     val resultScript = modifyInputFile(inputScript)
     placeholderFile.writeText(resultScript)
   }
 
   private fun getOutputDir(): String {
-    val let = getEnv(envName = "OUTPUT_DIR", defaultValue = "./")?.let(::File)?.absoluteFile
-      ?: throw IllegalStateException("OUTPUT_DIR not defined")
+    val let = getEnv(envName = OUTPUT_DIR_ENV, defaultValue = "./")?.let(::File)?.absoluteFile
```

Comment:
that change may break the Go part, do we need to change the variable names?

Reply:
I only extracted the value of the string to the const val, but the definition of env variable should not change and it should work with old environment